x-pack/solutions/security/plugins/cloud_security_posture/tsconfig.json

```diff
@@ -69,6 +69,7 @@
     "@kbn/code-editor-mock",
     "@kbn/core-saved-objects-utils-server",
     "@kbn/licensing-types",
+    "@kbn/logging-mocks",
```

Comment:
Not sure if we do really need to add that library for logging mock since we already have some tests that uses `loggingSystemMock` from `@kbn/core/server/mocks`. [example](https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/plugins/cloud_security_posture/server/create_indices/create_indices.test.ts). Can you check if reusing the existing would work instead?